.vscode/settings.json

```diff
@@ -18,6 +18,7 @@
         "UNDERCOLLATERALIZED"
     ],
     "python.analysis.extraPaths": [
-        "./src"
+        "./src",
+        "./examples"
```

Comment:
What is for? Asking for my knowledge

Reply:
It was auto added I think when I was running example to test new routes